sapphire/activation.py

```diff
@@ -29,7 +31,11 @@ def linsigmoid(x):
 
 def relu(x):
     """Rectified linear unit."""
-    return z3.z3.If(x > 0, x, 0)
+    return z3.If(x > 0, x, 0)
+    
+def lintanh(x):
```

Comment:
This should be a *variable* not a definition: `lin` returns a function, so just writing...
```python
lintanh = lin(math.tanh, -1.0, 1.0, 3, z3.If)
```
...would suffice.

Reply:
Ahh whoops! I changed it to be a variable now.